### PR TITLE
change Adelais En Ciel rev. 3 from STM32F303 to APM32F103 MCU

### PR DESCRIPTION
**Pull request by mechlovin**
## Description

change Adelais En Ciel rev. 3 from STM32F303 to APM32F103 MCU.
Thanks. 
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mechlovin/adelais/rgb_led/rev3/config.h [hunk 1]
```diff
@@ -52,13 +55,14 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
 #define DRIVER_LED_TOTAL 66
+
 #define RGB_MATRIX_KEYPRESSES // reacts to keypresses
 #define RGB_MATRIX_KEYRELEASES // reacts to keyreleases (instead of keypresses)
 #define RGB_MATRIX_FRAMEBUFFER_EFFECTS
 #define RGB_DISABLE_AFTER_TIMEOUT 0 // number of ticks to wait until disabling effects
-#define RGB_DISABLE_WHEN_USB_SUSPENDED // turn off effects when suspended
 #define RGB_MATRIX_LED_FLUSH_LIMIT 16 // limits in milliseconds how frequently an animation will update the LEDs. 16 (16ms) is equivalent to limiting to 60fps (increases keyboard responsiveness)
 #define RGB_MATRIX_MAXIMUM_BRIGHTNESS 255 // limits maximum brightness of LEDs to 200 out of 255. If not defined maximum brightness is set to 255
+#define RGB_DISABLE_WHEN_USB_SUSPENDED true // turn off effects when suspended
```
Comment (drashna) [inline keyboards/mechlovin/adelais/rgb_led/rev3/config.h:65]:
```suggestion
#define RGB_DISABLE_WHEN_USB_SUSPENDED // turn off effects when suspended
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/mechlovin/adelais/rgb_led/rev3/boards/adelais_f103/board.mk [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# List of all the board related files.
```
Comment (fauxpark) [inline keyboards/mechlovin/adelais/rgb_led/rev3/boards/adelais_f103/board.mk:1]: This file can be deleted.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/mechlovin/adelais/rgb_led/rev3/boards/adelais_f103/board.h [hunk 3]
```diff
@@ -0,0 +1,166 @@
+/*
```
[the remark this answers is not in the document]
Comment (mechlovin) [reply keyboards/mechlovin/adelais/rgb_led/rev3/boards/adelais_f103/board.h:1]: This file was deleted as instructed.

**Review by fauxpark — CHANGES_REQUESTED** [review]
Still some outstanding reviews here.

**Review by drashna — APPROVED** [review]

File: keyboards/mechlovin/adelais/rgb_led/rev3/mcuconf.h [hunk 4]
```diff
@@ -14,22 +14,13 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+
```
Comment (fauxpark) [inline keyboards/mechlovin/adelais/rgb_led/rev3/mcuconf.h:17]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by mechlovin** [comment]
@fauxpark would really appreciate your final review. Thanks a ton!

**Review by fauxpark — APPROVED** [review]

**Pull request merged**